modular_skyrat/modules/encounters/code/enclave_pirates.dm

```diff
@@ -205,13 +187,10 @@
 	<br> <span style=\"color:black;font-family:'Segoe Script';\"><p><b>Voronov Platon Aleksandrovich and Radich Katarina Dinovich.</b></p></span>"}
 
 /obj/machinery/suit_storage_unit/nri
-	suit_type = /obj/item/clothing/suit/space/hev_suit/nri/pirate
-	helmet_type = /obj/item/clothing/head/helmet/space/hev_suit/nri
+	mod_type = /obj/item/mod/control/pre_equipped/frontline/pirate
 	storage_type = /obj/item/tank/internals/oxygen/yellow
 
 /obj/machinery/suit_storage_unit/nri/captain
```

Comment:
what is the point of this

Reply:
I'm too lazy to make map edits to remove it in this PR; will be gone when I nerf the raider ship's cintents.